samples/modules/thrift/README.rst

```diff
@@ -0,0 +1,186 @@
+.. _sockets-echo-sample:
```

Comment:
I would put this in a sub directory, might start with one sample but might want to add another in future and having this in the root directory prevents that. Also the tag here is wrong